medicines/search-client/src/document_type.rs

```diff
@@ -55,6 +55,59 @@ impl Display for DocumentType {
     }
 }
 
+#[cfg_attr(
```

Comment:
May make more sense to have this in a new file (`territory_type/rs`?) or rename this file (can't think of a name right now that would cover both doc type and territory type).

Reply:
Will move this across. Lucky for me it's exposed through `models.rs` to the other projects in this repo so don't even have to change all of the imports 😅 .